<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IKApplicationClass</class>
 <widget class="QMainWindow" name="IKApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IKApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>38</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.005000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Iterations Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold Distance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>-180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zMax">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>-180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yMax">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>-180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xMax">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lamdaZ">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lamdaY">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lamdaX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IKApplication.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>zMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setZMin(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>changeComboboxItem(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setZMax(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setYMax(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setYMin(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setXMax(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setXMin(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lamdaZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setLamdaZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lamdaY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setLamdaY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lamdaX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IKApplicationClass</receiver>
   <slot>setLamdaX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeComboboxItem(int)</slot>
  <slot>setZMin(double)</slot>
  <slot>setYMin(double)</slot>
  <slot>setXMin(double)</slot>
  <slot>setZMax(double)</slot>
  <slot>setYMax(double)</slot>
  <slot>setXMax(double)</slot>
  <slot>setLamdaZ(double)</slot>
  <slot>setLamdaY(double)</slot>
  <slot>setLamdaX(double)</slot>
 </slots>
</ui>
